web/src/components/TtsPlayer.tsx

```diff
@@ -58,7 +59,7 @@ const BaseButton = styled(Button)`
   }
 `
 
-const PlayButton = styled(BaseButton)<{ disabled: boolean }>`
+const PlayButton = styled(IconButton)<{ disabled: boolean }>`
```

Comment:
:wrench:  `BaseButton` it has the shadow animation that give the button the feel if you were clicking a button:
```
  transition:
    box-shadow 0.2s ease,
    transform 0.1s ease;

  &:active {
    box-shadow: none;
    transform: translateY(2px);
  }
```